.github/workflows/push.yml

```diff
@@ -423,7 +423,7 @@ jobs:
           TAG: ${{ needs.set-version.outputs.tag }}
         run: |
           hub release download "${TAG}" -i '*windows_amd64*.zip'
-          printf "zip=%s" *.zip >> $GITHUB_OUTPUT
+          printf "::set-output name=zip::%s\n" *.zip
```

Comment:
Build MSI failuje a vůbec netuším proč. Tak revertuju změny co jsem tam udělal kvůli deprekování toho `set-output` a doufám že to zase fungovat začne. 😅 

<img width="1175" alt="image" src="https://user-images.githubusercontent.com/215660/196027420-edea23ec-6edc-4363-ae07-e65f41cc3c2c.png">

Reply:
(ked rozkliknes `run` v screenshote, tak tam by mali byt vsetky ENVs)